daemon/logger/fluentd/infomap.go

```diff
@@ -0,0 +1,48 @@
+package fluentd
+
+import (
+	"strings"
+
+	"github.com/docker/docker/daemon/logger"
+)
+
+const defaultInfoKeys = "container_id,container_name"
+
+// InfoMap returns information about a container as a map.
+func infoMap(info logger.Info) map[string]string {
```

Comment:
I don't think it can be more static than it is right now. It is built once per logger instance during `New()` similar to the `extra` attributes. The only improvement in this direction that comes to my mind is to combine both maps so that there is only one for loop in the `Log()` func.